orchestrator/rpc/api/events/event_api.go

```diff
@@ -8,6 +8,110 @@ import (
 	"github.com/pkg/errors"
 )
 
+// MinimalConsensusInfo
+func (api *PublicFilterAPI) MinimalConsensusInfo(ctx context.Context, requestedEpoch uint64) (*rpc.Subscription, error) {
+	notifier, supported := rpc.NotifierFromContext(ctx)
+	if !supported {
+		return &rpc.Subscription{}, rpc.ErrNotificationsUnsupported
+	}
+	rpcSub := notifier.CreateSubscription()
+
+	go func() {
+
+		batchSender := func(start, end uint64) error {
+			epochInfos, err := api.backend.ConsensusInfoByEpochRange(start)
+			if err != nil {
+				log.WithError(err).Error("Some epoch infos are missing in db.")
+				return errors.Wrap(err, "Missing epoch infos in db. Could not send over stream.")
+			}
+			latestFinalizedSlot := api.backend.LatestFinalizedSlot()
+			for _, ei := range epochInfos {
+				if err := notifier.Notify(rpcSub.ID, &generalTypes.MinimalEpochConsensusInfoV2{
+					Epoch:            ei.Epoch,
+					ValidatorList:    ei.ValidatorList,
+					EpochStartTime:   ei.EpochStartTime,
+					SlotTimeDuration: ei.SlotTimeDuration,
+					FinalizedSlot:    latestFinalizedSlot,
+				}); err != nil {
+					log.WithField("start", start).
+						WithField("end", end).
+						WithError(err).
+						Error("Failed to send epoch info. Could not send over stream.")
+					return errors.Wrap(err, "Failed to send epoch info. Could not send over stream.")
+				}
+				log.WithField("epoch", ei.Epoch).WithField("latestFinalizedSlot", latestFinalizedSlot).
+					Info("published epoch info to pandora")
+			}
+			return nil
+		}
+
+		startEpoch := requestedEpoch
+		endEpoch := api.backend.LatestEpoch()
+		if startEpoch <= endEpoch {
+			log.WithField("startEpoch", startEpoch).WithField("endEpoch", endEpoch).Debug("Sending previous epoch infos to pandora")
+			if err := batchSender(startEpoch, endEpoch); err != nil {
+				return
+			}
+		}
+
+		consensusInfo := make(chan *generalTypes.MinimalEpochConsensusInfoV2)
+		consensusInfoSub := api.events.SubscribeConsensusInfo(consensusInfo, requestedEpoch)
+		firstTime := true
+
+		for {
+			select {
+			case currentEpochInfo := <-consensusInfo:
+				log.WithField("epoch", currentEpochInfo.Epoch).
+					WithField("epochStartTime", currentEpochInfo.EpochStartTime).
+					Info("Sending consensus info to subscriber")
+
+				if firstTime {
+					firstTime = false
+					startEpoch = endEpoch
+					endEpoch = api.backend.LatestEpoch()
+
+					if startEpoch+1 < endEpoch {
+						log.WithField("startEpoch", startEpoch).WithField("endEpoch", endEpoch).
+							Debug("successfully published left over epoch infos")
```

Comment:
Inconsistent log formatting
```suggestion
							Debug("Successfully published left over epoch infos")
```